app/views/play/level/PlayLevelView.coffee

```diff
@@ -190,6 +190,10 @@ module.exports = class PlayLevelView extends RootView
     @listenToOnce @levelLoader, 'world-necessities-loaded', @onWorldNecessitiesLoaded
     @listenTo @levelLoader, 'world-necessity-load-failed', @onWorldNecessityLoadFailed
 
+  checkInClan: (clans) ->
```

Comment:
Naming is clearer to me like this. This function could also be inlined if we want.
```suggestion
  hasAccessThroughClan: (level) ->
```